src/libs/OptionsListUtils.ts

```diff
@@ -1684,13 +1681,13 @@ function isValidReport(reportOption: SearchOption<Report>, config: GetValidRepor
     const isMoneyRequestReport = option.isMoneyRequestReport;
     const isSelfDM = option.isSelfDM;
     const isChatRoom = option.isChatRoom;
-    const accountIDs = getParticipantsAccountIDsForDisplay(report);
+    const accountIDs = getParticipantsAccountIDsForDisplay(option.item);
 
     if (excludeNonAdminWorkspaces && !isPolicyAdmin(option.policyID, policies)) {
         return false;
     }
 
-    if (isPolicyExpenseChat && report?.isOwnPolicyExpenseChat && !includeOwnedWorkspaceChats) {
+    if (isPolicyExpenseChat && option.isOwnPolicyExpenseChat && !includeOwnedWorkspaceChats) {
```

Comment:
Shouldnt this be option.item?
```suggestion
    if (isPolicyExpenseChat && option.item.isOwnPolicyExpenseChat && !includeOwnedWorkspaceChats) {
```

Reply:
I think, we're safe to have `option.isOwnPolicyExpenseChat`. Look at a screen which shows:
```
  console.log('morwa option, option.item', option.isOwnPolicyExpenseChat, option.item.isOwnPolicyExpenseChat);
  ```

<img width="337" height="126" alt="image" src="https://github.com/user-attachments/assets/9a470e01-31da-4394-8c64-04ced1a2d560" />